sdk/openai/openai/README.md

```diff
@@ -250,6 +252,52 @@ async function main(){
 }
 ```
 
+### Analyze Business Data
+
+This example generates chat responses to input chat questions about your business data. The business data is provided through an Azure Cognitive Search index. To learn more about how to setup an Azure Cognitive Search index as a data source, see [Quickstart: Chat with Azure OpenAI models using your own data][msdocs_quickstart_byod].
+
+
+```javascript
+const { OpenAIClient } = require("@azure/openai");
+
+async function main(){
+  const endpoint = "https://myaccount.openai.azure.com/";
+  const client = new OpenAIClient(endpoint, new DefaultAzureCredential());
+
+  const deploymentId = "gpt-35-turbo";
```

Comment:
Because only model version `0301` is supported for gpt-35-turbo, maybe we should show that one in this example

Reply:
This is the deployment ID not the model name. I didn't want to mention any specific limitations here because they could get out of date fast.